### PR TITLE
[JENKINS-64107] Convert SSHD module to a plugin

### PR DESCRIPTION
**Pull request by kuisathaverat**
[JENKINS-55582](https://issues.jenkins-ci.org/browse/JENKINS-55582) Convert modules to plugins
[JENKINS-64107](https://issues.jenkins-ci.org/browse/JENKINS-64107) Convert SSHD module to a plugin
[JEP](https://github.com/jenkinsci/jep/pull/326) Convert modules to plugins

* This PR change to use the  sshd-plugin
* Bump the Core to the 2.282

### CODE REVIEW

**Comment by kuisathaverat** [comment]
waiting for the core 2.282 to be released

File: pom.xml [hunk 1]
```diff
@@ -68,6 +68,7 @@
         <dependency>
             <groupId>org.jenkins-ci.modules</groupId>
             <artifactId>sshd</artifactId>
+            <version>3.0.1</version>
```
Comment (kuisathaverat) [inline pom.xml:71]:
```suggestion
            <version>3.0.3</version>
```

**Comment by timja** [comment]
@olivergondza

**Comment by olivergondza** [comment]
Does it really require depending on such a bleeding edge Jenkins core? In that case, I suggest postponing incorporating this.

**Review by olivergondza — APPROVED** [review]

**Comment by timja** [comment]
It does yes, it’s been detached from Jenkins core, there’s a major update of the lib coming in the detached plugin and I believe all consumers need to update or it’s best if they do at least

Updating the version shouldn’t really be an issue?  Just cut a release of anything pending and then if something super important comes along then backport it to the previous line?

**Comment by olivergondza** [comment]
> Updating the version shouldn’t really be an issue? Just cut a release of anything pending and then if something super important comes along then backport it to the previous line?

Yep, that would be the process. I do not feel strongly.

**Pull request merged**